job/validators.py

```diff
@@ -352,12 +367,15 @@ def cross_clean_pv(pvmodel):
         ElectricLoad
         If user does not provide values, set defaults conditional on off-grid flag
         """
-
-        if self.models["ElectricLoad"].__getattribute__("critical_load_pct") == None:
-            if self.models["Settings"].off_grid_flag==False:
-                self.models["ElectricLoad"].critical_load_pct = 0.5
-            else:
+        '''
```

Comment:
Similar comment to above. If we're just overriding in the pff grid situation anyways, we can just make the Django default the on grid default (0.5) and delete the else block below

Reply:
Implemented in https://github.com/NREL/REopt_API/pull/343/commits/152ac98842eaaf16a7068f0412b34b0893f0886a